settings.php

```diff
@@ -0,0 +1,37 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Implementaton of the quizaccess_honestycheck plugin.
+ *
+ * @package   quizaccess_honestycheck
+ * @author     Sumaiya Javed <sumaiya.javed@catalyst.net.nz>
+ * @copyright 2011 The Open University
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die;
+
+$settings->add(new admin_setting_configselect(
+    'quizaccess_honestycheck/honestycheckrequired',
+    get_string('honestycheckrequired', 'quizaccess_honestycheck'),
+    get_string('honestycheckrequired', 'quizaccess_honestycheck'),
+    1,
```

Comment:
Making the default 1 might be what you want, but you are contributing to an open source project. For most people, the default of 0 is more appropriate, so we should set that.